apps/apps.humanatlas.io/src/app/components/server-selector/server-selector.component.scss

```diff
@@ -0,0 +1,41 @@
+:host {
+  display: block;
+  width: 100%;
+
+  mat-form-field {
+    width: 100%;
+  }
+
+  .selected-server-description {
+    font: var(--mat-sys-label-small);
+    color: var(--schemes-primary-p-40-gray-2, #4c4c58);
```

Comment:
I suspect you copied the `var(...)` from figma. Sadly those are incorrect. The correct var is `--mat-sys-primary`. Also you don't need the fallback value since they will always be defined. Please replace all `--schemes-...` with their correct equivalent, i.e. `--mat-sys-primary`, `--mat-sys-secondary`, `--mat-sys-secondary-fixed`, etc.